### DISCUSSION TITLE
Any templates to buy that are using chakra UI?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [chakra-ui/chakra-ui#7824] [General] [answered]
Hi all, do you know any chakra ui ready to use templates to buy? Thanks.

**Comment by coopbri** [comment, marked as answer]
- [Chakra Pro](https://pro.chakra-ui.com/components) is marketed as a collection of "components", but in reality they are essentially templates composing the Chakra UI components.
- There are also free (MIT-licensed) templates at https://chakra-templates.dev
- [Saas UI](https://saas-ui.dev) is a library that composes Chakra UI, and has both free and paid (Pro) components
- Creative Tim has a collection of full [Chakra UI application templates](https://www.creative-tim.com/search?q=chakra) with free and paid variants

Hope that helps

**Reply by Abdul-Rouf-Shaik** [reply]
Great resources, Thanks for sharing

**Reply by thesohailjafri** [reply]
You could also check.

![image](https://github.com/user-attachments/assets/80c62c3a-51fc-4c8d-acbb-7782288c745a)


https://chakraframer.com/
and https://chakraframer.com/collections it contains collections of freemium components they are pretty handy.

I personally liked this https://chakraframer.com/components/hover-overlapping-text used it on https://withsam.in/ for connect with me CTA

**Reply by Sahil-Chaubey** [reply]
I used the encryption button template from your website on my portfolio, and it looks absolutely amazing! Super useful—thank you!
